### ISSUE TITLE
[Pyload-ng] module pyload.core.config not found

### ISSUE THREAD

**Issue by Galiley** [pyload/pyload#3540]
### Describe the problem

The cli failed on Python 3.8 because of an import error

### Debug log

```
$pyload
ModuleNotFoundError: No module named 'pyload.core.config'
Traceback (most recent call last):
  File "/home/galiley/.local/bin/pyload", line 8, in <module>
    sys.exit(main())
  File "/home/galiley/.local/lib/python3.8/site-packages/pyload/script.py", line 124, in main
    run(core_args, args.daemon)
  File "/home/galiley/.local/lib/python3.8/site-packages/pyload/script.py", line 108, in run
    pyload_core = Core(*core_args)
  File "/home/galiley/.local/lib/python3.8/site-packages/pyload/core/__init__.py", line 91, in __init__
    self._init_config(userdir, cachedir, storagedir, debug)
  File "/home/galiley/.local/lib/python3.8/site-packages/pyload/core/__init__.py", line 107, in _init_config
    from .config.parser import ConfigParser
ModuleNotFoundError: No module named 'pyload.core.config'
```

**Comment by Ch3nYe** [comment]
python3.6.9 also encounter this error：
`ModuleNotFoundError: No module named 'pyload.core.config'
`

**Comment by AlexanderProd** [comment]
I have the same problem with `Python 3.7.2`

```
ModuleNotFoundError: No module named 'pyload.core.config'
Traceback (most recent call last):
  File "/home/pi/.local/bin/pyload", line 8, in <module>
    sys.exit(main())
  File "/home/pi/.local/lib/python3.7/site-packages/pyload/script.py", line 124, in main
    run(core_args, args.daemon)
  File "/home/pi/.local/lib/python3.7/site-packages/pyload/script.py", line 108, in run
    pyload_core = Core(*core_args)
  File "/home/pi/.local/lib/python3.7/site-packages/pyload/core/__init__.py", line 91, in __init__
    self._init_config(userdir, cachedir, storagedir, debug)
  File "/home/pi/.local/lib/python3.7/site-packages/pyload/core/__init__.py", line 107, in _init_config
    from .config.parser import ConfigParser
ModuleNotFoundError: No module named 'pyload.core.config'
```

**Comment by serl** [comment]
Same here with Python 3.6.9 and `pyload-ng==0.5.0a5.dev545`

**Comment by kuipumu** [comment]
Same issue using Python 3.6.10 and pyload-ng==0.5.0a5.dev545

**Comment by malikbenkirane** [comment]
Also with
```
> pyload --version
pyLoad 0.5.0
```

**Comment by malikbenkirane** [comment]
@Galiley could you rephrase the title of the issue to something like "module pyload.core.config not found" modulo guidelines.
If we want to get some attention from contributors as there is so much issues open at this time. With some technical terms in the title I hope this will increase the odds some developer find out about the issue with ease.

**Comment by drt1** [comment]
Same here, any news on this?

**Comment by JoeDevler** [comment]
Same issue.
Python 3.8.2
pyLoad 0.5.0

**Comment by msdos** [comment]
This is a duplicate of https://github.com/pyload/pyload/issues/3428.

Check https://github.com/pyload/pyload/issues/3428#issuecomment-522275713 for instructions.
